src/core_plugins/metric_vis/public/metric_vis_params.html

```diff
@@ -0,0 +1,206 @@
+        </div>
+
+        <div
+          ng-click="addRange()"
+          class="kuiButton kuiButton--primary kuiButton--fullWidth">
+          Add Range
+        </div>
+      </div>
+    </div>
+  </div>
+  <div ng-show="vis.params.metric.colorsRange.length > 1">
+    <div class="kuiSideBarCollapsibleTitle">
+      <div
+        kbn-accessible-click
+        aria-expanded="{{!!showColorOptions}}"
+        aria-controls="metricOptionsColors"
+        aria-label="Toggle color options"
+        class="kuiSideBarCollapsibleTitle__label"
+        ng-click="showColorOptions = !showColorOptions"
+      >
+      <span
+        aria-hidden="true"
+        ng-class="{
+        'fa-caret-down': showColorOptions,
+        'fa-caret-right': !showColorOptions
+        }"
+        class="fa fa-caret-right kuiSideBarCollapsibleTitle__caret"
+      ></span>
+        <span class="kuiSideBarCollapsibleTitle__text">
+        Color Options
+      </span>
+      </div>
+    </div>
+    <div id="metricOptionsColors" ng-if="showColorOptions" class="kuiSideBarCollapsibleSection">
+      <div class="kuiSideBarSection">
+        <div class="kuiSideBarFormRow">
+          <label class="kuiSideBarFormRow__label" for="colorSchema">
+            Color Schema
+          </label>
+          <div class="kuiSideBarFormRow__control">
+            <select
+              id="colorSchema"
+              class="kuiSelect kuiSideBarSelect"
+              ng-model="vis.params.metric.colorSchema"
+              ng-options="mode for mode in collections.colorSchemas"
+            ></select>
+          </div>
+          <div class="text-info text-center" ng-show="customColors" ng-click="resetColors()">reset colors</div>
+        </div>
+
+        <div class="kuiSideBarFormRow">
+          <label class="kuiSideBarFormRow__label" for="invertColors">
+            Reverse Color Schema
+          </label>
+          <div class="kuiSideBarFormRow__control">
+            <input class="kuiCheckBox" id="invertColors" type="checkbox" ng-model="vis.params.metric.invertColors">
+          </div>
+        </div>
+      </div>
+    </div>
+  </div>
+
+  <div>
+    <div class="kuiSideBarCollapsibleTitle">
+      <div
+        kbn-accessible-click
+        aria-expanded="{{!!showStyle}}"
+        aria-controls="metricOptionsStyle"
+        aria-label="Toggle style options"
+        class="kuiSideBarCollapsibleTitle__label"
+        ng-click="showStyle = !showStyle"
+      >
+        <span
+          aria-hidden="true"
+          ng-class="{
+          'fa-caret-down': showStyle,
+          'fa-caret-right': !showStyle
+          }"
+          class="fa fa-caret-right kuiSideBarCollapsibleTitle__caret"
+        ></span>
+        <span class="kuiSideBarCollapsibleTitle__text">
+          Style
+        </span>
+      </div>
+    </div>
+    <div id="metricOptionsStyle" ng-if="showStyle" class="kuiSideBarCollapsibleSection">
+
+      <div class="kuiSideBarSection">
+        <div class="kuiSideBarFormRow">
```

Comment:
I agree with @alexfrancoeur that we should not have this options then.

For now (6.0), can we hide the color options?

`<div class="kuiSideBarFormRow" ng-show="vis.params.metric.colorsRange.length > 1">`

For a next minor, and I agree we should not wait on this, we can reintroduce the legend.